cms/lib/xblock/tagging/tagging.py

```diff
@@ -71,25 +74,28 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
         """
         Handler to save choosen tags with connected XBlock
         """
-        found = False
-        if 'tag' not in request.params:
-            return Response("The required parameter 'tag' is not passed", status=400)
+        posted_data = request.params.dict_of_lists()
+        saved_tags = {}
 
-        tag = request.params['tag'].split(':')
+        need_update = False
 
         for av_tag in self.get_available_tags():
-            if av_tag.name == tag[0]:
-                if tag[1] == '':
-                    self.saved_tags[tag[0]] = None
-                    found = True
-                elif tag[1] in av_tag.get_values():
-                    self.saved_tags[tag[0]] = tag[1]
-                    found = True
-
-        if not found:
-            return Response("Invalid 'tag' parameter", status=400)
-
-        return Response()
+            tag_key = '%s[]' % av_tag.name
+            if tag_key in posted_data and len(posted_data[tag_key]) > 0:
```

Comment:
Replace `and len(posted_data[tag_key]) > 0` with just `and posted_data[tag_key])`